capa/features/extractors/binja/function.py

```diff
@@ -5,6 +5,7 @@
 # Unless required by applicable law or agreed to in writing, software distributed under the License
 #  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and limitations under the License.
+import logging
 import sys
```

Comment:
```suggestion
import sys
import logging
```

Reply:
I ended up removing the `logging` from imports since it is not used at all. It was only used when I troubleshoot the issue. And its usage was removed once I get it fixed